src/Cli.Tests/EndToEndTests.cs

```diff
@@ -668,7 +668,12 @@ public void TestHelpWriterOutput(string command, string flags, string[] expected
         Assert.IsNotNull(output);
 
         // Version Info logged by dab irrespective of commands being parsed correctly.
-        StringAssert.Contains(output, $"{Program.PRODUCT_NAME} {ProductInfo.GetProductVersion()}", StringComparison.Ordinal);
+        StringAssert.Contains(ProductInfo.GetProductVersion(), $"{Program.PRODUCT_NAME} {ProductInfo.GetProductVersion()}", StringComparison.Ordinal);
```

Comment:
```suggestion
        StringAssert.Contains(output, $"{Program.PRODUCT_NAME} {ProductInfo.GetProductVersion()}", StringComparison.Ordinal);
```

Keep the test string to check the contents of to be the output from the dab command itself, otherwise this assert is not testing anything - infact it would fail coz the first string will then just be a subset of the second..

Reply:
Good point :)